tools/pvacvector/run.py

```diff
@@ -454,8 +454,10 @@ def main(args_input=sys.argv[1:]):
         VectorVisualization(results_file, base_output_dir, args.spacers).draw()
 
     if not args.keep_tmp_files:
-        shutil.rmtree(os.path.join(base_output_dir, 'MHC_Class_I'), ignore_errors=True)
-        shutil.rmtree(os.path.join(base_output_dir, 'MHC_Class_II'), ignore_errors=True)
+        for subdirectory in range(tries+1):
```

Comment:
It looks like `tries` already gets incremented right before we exit the `while` loop (line 443). Is it necessary to do so again here?